conf/config-default.yaml

```diff
@@ -303,6 +302,10 @@ nginx_config:                     # config for render the template to generate n
 #       path: "logs/consul_kv.dump"
 #       expire: 2592000            # unit sec, here is 30 day
 #  kubernetes:
+#    ### kubernetes service discovery both support single-cluster and multi-cluster mode
+#    ### applicable to the case where the service is distributed in a single or multiple kubernetes clusters.
+#
+#    ### Single-Cluster Mode ###
```

Comment:
Hi, I see `single-cluster` appearing multiple times in this article. Please keep the same way of writing, either `single-cluster` or `Single-Cluster`.